tests/unit/test-util/test.json.js

```diff
@@ -0,0 +1,27 @@
+import {it, describe} from 'mocha';
+import {assert} from 'chai';
+import {fs} from 'mz';
+
+import {parseJSON} from '../../../src/util/json';
+import {basicManifest} from '../helpers';
+
+
+describe('util.parseJSON', () => {
+
+  it('returns a parsed JSON', () => {
+    const parsedJSON = parseJSON(JSON.stringify(basicManifest));
+    assert.deepEqual(parsedJSON, basicManifest);
+  });
+
+  it('returns a parsed JSON from file with BOM Chanracter', () => {
+    const dummyJson = { manifest_version: 2 };
+    fs.readFile('./test.manifest.json', function read(err, data) {
```

Comment:
because you're using `fs` from `mz`, it returns a promise. You can check out other examples of `fs.readFile()` in the test suite to see how to use it as a promise.